be/src/exec/exec_node.h

```diff
@@ -214,6 +219,13 @@ class ExecNode {
 
     vectorized::RuntimeFilterProbeCollector& runtime_filter_collector() { return _runtime_filter_collector; }
 
+    // local runtime filters that are conducted on this ExecNode are planned by FE.
+    std::set<TPlanNodeId> local_rf_waiting_set() { return _local_rf_waiting_set; }
```

Comment:
is this method `const` or not? could return reference?

Reply:
ok